templates/terraform/resource.erb

```diff
@@ -247,6 +247,14 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
         return fmt.Errorf("Error waiting to create <%= object.name -%>: %s", err)
     }
 
+    <% object.gettable_properties.each do |prop| -%>
+    <% if object.identity.include?(prop) -%>
+    if err := d.Set("<%= prop.name.underscore -%>", flatten<%= resource_name -%><%= titlelize_property(prop) -%>(opRes["<%= prop.api_name -%>"], d, config)); err != nil {
```

Comment:
This uses opRes, which is set below, I think. Can you explain how this change is safe?

Reply:
It's actually set in line 239, just above.  The code below modifies it, and we need to not use that modified version.

Reply:
Is the same true in other resources? I think this makes spanner use the full `name` instead of the short `name`, for example.

Reply:
Ah, you're right about that!  :( I'll see what I can do.